encoder/vaapiencoder_base.cpp

```diff
@@ -430,6 +474,17 @@ bool VaapiEncoderBase::ensureMiscParams (VaapiEncPicture* picture)
         if (frameRate)
             fill(frameRate);
     }
+
+    uint32_t numROI = m_videoParamCommon.roiParams.numROI;
+    if (numROI != 0) {
```

Comment:
we perfer if (!numROI) see https://webkit.org/code-style-guidelines/#zero-comparison for details